examples/wrapped-certs/example.0.json

```diff
@@ -1,25 +1,25 @@
 {
```

Comment:
dont push changes for that file, I dunno how to handle it for the moment

Reply:
So i should add it to git ignore and replace with the original one?

Reply:
can either add the file to git ignore and/or just remove from this commit

dont delete the file I think it's better so that can show examples with unwrapped / wrapped document